lib/venue.rb

```diff
@@ -1,2 +1,30 @@
 class Venue
+  attr_reader :name, :capacity, :patrons
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+  def add_patron(name)
+    @patrons << name
+  end
+  def yell_at_patrons
+    @patrons.map! { |name| name.upcase }
+  end
+
+  def over_capacity?
+    if @patrons.size <= @capacity
+      return false
+    elsif @patrons.size > @capacity
+      return true
+    else
+      return "Invalid input"
+    end
+  end
+  def kick_out
+    while @patrons.size > @capacity
```

Comment:
Nice use of a `while` loop. Is there a method that is already checking whether we are over_capacity or not?? Could you use that method here?